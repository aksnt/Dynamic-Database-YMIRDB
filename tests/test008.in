SET a 1 2
SNAPSHOT
SET b 4 5 a
SNAPSHOT
PURGE a
PURGE b
BYE
