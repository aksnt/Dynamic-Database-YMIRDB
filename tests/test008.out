> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> not permitted

> ok

> bye
